README.md

```diff
@@ -44,9 +47,10 @@ This guides some of its principles:
 - It should be possible to do everything that isn't drawing just with keys.
   Ideally, only with the keys on one half of the keyboard, to make presentations flow smoothly.
   You shouldn't need to move your mouse all the way to the left to change tools, or to move your hand to the right to switch to the pen tool.
-- By default, it has pages, rather than extending in different directions.
+- It has pages, rather than extending in different directions.
   It should feel like writing on multiple blackboards, and not an infinite sheet of paper.
-- Pages are fixed at a 16:9 ratio, so when they're later saved to a PDF, it's in the same dimensions as a slideshow.
+  - We are considering changing this to allow an infinite scroll mode (see [#6](https://github.com/cjquines/qboard/issues/6#issue-688305306)).
+- Pages are fixed at a 16:9 ratio, so when they're later exported to a PDF, it's in the same dimensions as a slideshow.
```

Comment:
Do we write something additional about how in full screen mode, it will typically fill the whole screen (since most screens are 16:9)

And on a tangential note, should aspect ratio become a changeable setting (yes, i know dev mode lets people hack this, but that's not user-friendly)

Reply:
unclear

tangent: maybe, but this seems less critical to me. consider e.g. google jamboard

Reply:
i think it's probably worth adding then

tangent:
i'm not sure jamboard is the best example; their app is certainly designed for hypersimplicity (they don't even have lines), and the size isn't locked because that's what the presentation width is (we know this because they have a topbar).

i think better, though not perfect, comparisons are google drawings, google slides, and powerpoint. as an aside, i don't think jamboard should dictate our design decisions because you can't even select strokes

Reply:
Made change in ecec882, feel free to revert

Reply:
argument against: maybe considered features should just be all in issues, because they currently are all there, rather than in readme. idk

Reply:
I don't entirely agree; readers have no clue _which_ features/PRs are important (also this is an argument for labelling app changes differently from code changes with no app effects), and—even if they did—they wouldn't get these short curated summaries.